### PR TITLE
changed installation method to module based

### PR DESCRIPTION
**Pull request by tom-rosewell**

### CODE REVIEW

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 1]
```diff
@@ -54,11 +54,11 @@ The Couchbase Capella free trial version comes with the Travel Sample Bucket, an
 == Quick Installation
 
 Version 2 of the Go SDK has added support for https://github.com/golang/go/wiki/Modules[Go Modules]. 
-You can use `go get` to download the SDK:
+You can install the Go SDK by adding the following to your module's `go.mod` file:
```
Comment (chvck) [inline modules/hello-world/pages/start-using-sdk.adoc:57]: I don't think that this is very idiomatic for Go. Typically users would run `go get github.com/couchbase/gocb/v2`, and that automatically adds the dependency to the go.mod file. If we want to tie it to a version we could use `go get github.com/couchbase/gocb/v2@v2.6.1` or `go get github.com/couchbase/gocb/v2@latest` although the latte is pretty redundant.

**Review by chvck — CHANGES_REQUESTED** [review]

File: modules/hello-world/pages/start-using-sdk.adoc (hunk 1 above)
Comment (tom-rosewell) [reply modules/hello-world/pages/start-using-sdk.adoc:57]: Do you think it would be best to include both, or keep it the way it is (with go get)?

**Comment by tom-rosewell** [comment]
The entire testing framework is quite broken, I am working on fixing it

File: modules/hello-world/pages/start-using-sdk.adoc (hunk 1 above)
Comment (chvck) [reply modules/hello-world/pages/start-using-sdk.adoc:57]: Probably keep it the way that it is. I don't think that any users will be manually adding it to the go.mod.

Comment (tom-rosewell) [reply modules/hello-world/pages/start-using-sdk.adoc:57]: Keeping as it, closing the issue

**Pull request closed without merging**